hlint.cabal

```diff
@@ -54,6 +54,8 @@ library
         base == 4.*, process, filepath, directory, containers,
         unordered-containers, vector, text, bytestring,
         transformers,
+        file-embed,
+        utf8-string,
```

Comment:
Where is the utf8-string dependency coming from?

Reply:
embed stores a `ByteString` of the UTF-8 data files, so we have to convert it back.